inquire/src/prompts/multicount/mod.rs

```diff
@@ -0,0 +1,427 @@
+mod action;
+mod config;
+mod prompt;
+#[cfg(test)]
+#[cfg(feature = "crossterm")]
+mod test;
+
+pub use action::*;
+
+use std::fmt::Display;
+
+use crate::{
+    config::get_configuration,
+    error::{InquireError, InquireResult},
+    formatter::MultiCountFormatter,
+    list_option::ListOption,
+    prompts::prompt::Prompt,
+    terminal::get_default_terminal,
+    type_aliases::Scorer,
+    ui::{Backend, MultiCountBackend, RenderConfig},
+    validator::MultiOptionValidator,
+};
+
+use self::prompt::MultiCountPrompt;
+
+#[cfg(feature = "fuzzy")]
+use fuzzy_matcher::{skim::SkimMatcherV2, FuzzyMatcher};
+#[cfg(feature = "fuzzy")]
+use once_cell::sync::Lazy;
+#[cfg(feature = "fuzzy")]
+static DEFAULT_MATCHER: Lazy<SkimMatcherV2> = Lazy::new(|| SkimMatcherV2::default().ignore_case());
+type CountedListOption<T> = (u32, ListOption<T>);
```

Comment:
Looks like we're not using this type on all prompt functions returning the answer